static/js/chart/types.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import { Point } from "../topic_page/ranking_unit";
+import { Point } from "../types/ranking_unit_types";
```

Comment:
While we're here... rename Point to RankingPoint?  (we have so many types of points...)